include/rocksdb/statistics.h

```diff
@@ -518,6 +518,11 @@ enum Tickers : uint32_t {
   // ReadOptions.auto_readahead_size is set.
   READAHEAD_TRIMMED,
 
+  // Number of FIFO compactions that drop files based on different reasons
+  FIFO_MAX_SIZE_COMPACTIONS,
+  FIFO_TTL_COMPACTIONS,
+  FIFO_CHANGE_TEMPERATURE_COMPACTIONS,
```

Comment:
Is "drop files" an accurate way to describe this type of compaction? I thought it's a time-triggered intra-L0 that rewrites data into the next tier

Reply:
You are correct - we should not include FIFO_CHANGE_TEMPERATURE_COMPACTIONS as deletion compaction. Removed ticker for it.